### PR TITLE
Rol accumulator

### PR DESCRIPTION
**Pull request by sa12123**
Implemented opcode RolAccumulator
(also fixed: Opcode 0x46 should be LsrZeropage, not LsrAccumulator)

### CODE REVIEW

File: core/src/opcode.cpp [hunk 1]
```diff
@@ -254,6 +256,7 @@ MemoryAccess get_memory_access(const Family family) {
     case Family::DEY:
     case Family::DEX:
     case Family::EOR:
+    case Family::ROL:
```
Comment (johnor) [inline core/src/opcode.cpp]: It doesn't really matter for implied/accumulator that you implemented in this PR, but ROL is a read/write instruction so it should return ReadWrite here.

Comment (sa12123) [reply core/src/opcode.cpp]: Ok, thanks! Will fix!

**Comment by sa12123** [comment]
Changed according to comment.

**Comment by robinlinden** [comment]
Rebased and squashed the review commit.

**Pull request merged**